.travis.yml

```diff
@@ -22,14 +22,19 @@ before_script:
   - git config --global user.name "Berkshelf"
 matrix:
   include:
-    - rvm: 2.5.7
+    - rvm: 2.7.1
     - rvm: 2.6.5
+      before_install:
+        - gem install bundler:2.1.4 -N
     - rvm: ruby-head
     - rvm: 2.6.5
       before_install:
+        - gem install bundler:2.1.4 -N
         - gem update --system
         - rm Gemfile.lock
     - rvm: 2.6.5
```

Comment:
most of these should be updated to 2.7.1 where possible, ideally there should only be one 2.6.5 that runs the rake test for backcompat.  really that job should pin to chef-15 as well (thinking that through, i'm also kind of thinking that the Gemfile.lock in this repo should be deleted).